tests/bgp/test_4-byte_asn_community.py

```diff
@@ -0,0 +1,231 @@
+    logger.info("Neighbor BGP Config: {}".format(nbrhosts[neigh]["host"].shell("show run bgp")['stdout']))
+    logger.info('Setup_info: {}'.format(setup_info))
+
+    yield setup_info
+
+    # restore config to original state
+    config_reload(duthost)
+    config_reload(neighbors[neigh], is_dut=False)
+
+    # verify sessions are established
+    bgp_facts = duthost.bgp_facts(instance_id=asic_index)['ansible_facts']
+    for k, v in bgp_facts['bgp_neighbors'].items():
+        if v['description'].lower() not in skip_hosts:
+            logger.debug(v['description'])
+            assert v['state'] == 'established'
+
+
+def test_4_byte_asn_community(setup):
+    cmd = 'vtysh -n {} \
+    -c "config" \
+    -c "no router bgp {}" \
+    -c "router bgp {}" \
+    -c "bgp router-id {}" \
+    -c "bgp log-neighbor-changes" \
+    -c "no bgp ebgp-requires-policy" \
+    -c "no bgp default ipv4-unicast" \
+    -c "bgp bestpath as-path multipath-relax" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "address-family ipv4 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V4 in" \
+    -c "neighbor {} route-map TO_BGP_PEER_V4 out" \
+    -c "neighbor {} activate" \
+    -c "maximum-paths 64" \
+    -c "exit-address-family" \
+    -c "address-family ipv6 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V6 in" \
+    -c "neighbor {} route-map TO_BGP_PEER_V6 out" \
+    -c "neighbor {} activate" \
+    -c "maximum-paths 64" \
+    -c "exit-address-family" \
+    '.format(setup['namespace'], setup['dut_asn'], dut_4byte_asn, setup['dut_bgp_id'],
+             setup['peer_group_v4'], setup['peer_group_v6'], setup['neigh_ip_v4'], neighbor_4byte_asn,
+             setup['neigh_ip_v4'], setup['peer_group_v4'], setup['neigh_ip_v4'], setup['neigh'], setup['neigh_ip_v4'],
+             setup['neigh_ip_v4'], setup['neigh_ip_v6'], neighbor_4byte_asn, setup['neigh_ip_v6'],
+             setup['peer_group_v6'], setup['neigh_ip_v6'], setup['neigh'], setup['neigh_ip_v6'], setup['neigh_ip_v6'],
+             setup['dut_ipv4_network'], setup['peer_group_v4'], setup['peer_group_v4'], setup['peer_group_v4'],
+             setup['neigh_ip_v4'], setup['dut_ipv6_network'], setup['peer_group_v6'], setup['peer_group_v6'],
+             setup['peer_group_v6'], setup['neigh_ip_v6'])
+    logger.debug(setup['duthost'].shell(cmd, module_ignore_errors=True))
+
+    cmd = 'vtysh \
```

Comment:
by using above `run_vtysh`, this 2nd `cmd` is not needed

Reply:
can you add the `namespace` or `cli_options` for the neighbor as well, T1 neighbor could be multi-asic as well

Reply:
I added neigh_cli_options for the neighbor command.